src/deprecatedCompat/5.0/isParameter.ts

```diff
@@ -0,0 +1,20 @@
+import {
+    Node,
+    ParameterDeclaration,
+    SyntaxKind
+} from "../_namespaces/ts";
+import { deprecate } from "../deprecate";
+
+// DEPRECATION: Renamed node tests
+// DEPRECATION PLAN:
+//     - soft: 5.0
+//     - warn: 5.1
+//     - error: TBD

```

Comment:
These values are made up and are up for discussion.

Reply:
Yeah....`SyntaxKind.Parameter` is used quite a bit right now. We should probably have a very long warn period.

Reply:
This comment is of course out of date, this missed 5.0 which was a shame.

I'm not totally sure what the plan here is anymore. I will note that we actually can't "remove" an enum item; there's not really a deprecation helper mechanism for that. So most people would work forever, it's just those who are using `isParameter` that will have a problem.